cmd/opentelemetry/app/defaultconfig/default_config.go

```diff
@@ -71,11 +73,7 @@ func (c ComponentSettings) CreateDefaultConfig() (*configmodels.Config, error) {
 		return nil, err
 	}
 	receivers := createReceivers(c.ComponentType, c.ZipkinHostPort, c.Factories)
-	processors := configmodels.Processors{}
-	resProcessor := c.Factories.Processors["resource"].CreateDefaultConfig().(*resourceprocessor.Config)
-	if len(resProcessor.Labels) > 0 {
-		processors[resProcessor.Name()] = resProcessor
-	}
+	processors := createProcessors(c.Factories)
```

Comment:
Could this method return the list of processor names aswell, rather than having a hardcoded list on line 89.

Reply:
+1 that is better